bindings/pyroot_experimental/PyROOT/CMakeLists.txt

```diff
@@ -86,9 +86,9 @@ ROOT_STANDARD_LIBRARY_PACKAGE(ROOTPython
 )
 
 file(COPY python/ROOT DESTINATION ${py_localruntimedir})
-install(DIRECTORY python/ROOT DESTINATION ${runtimedir})
+install(DIRECTORY python/ROOT DESTINATION ${CMAKE_INSTALL_PYROOTDIR})
 
-set(d $ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/${runtimedir})
+set(d $ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR})
```

Comment:
two cases: the user specifies an absolute path for CMAKE_INSTALL_PYROOTDIR or we have a relative path, which may come from the user or from us (default case).
when it's an absolute path, don't prepend the CMAKE_INSTALL_PREFIX